File: Exercitar java/Exercicio02/Exercicio02.java




import java.util.Scanner;


class Main {
    public static void main(String[] args) throws Exception { 
        //Leitura dos dados 
        System.out.println("Insira o coeficiente a:");
        int a = EntradaTeclado.leInt();
        
        System.out.println("Insira o coeficiente b:");
        int b = EntradaTeclado.leInt();

        System.out.println("Insira o coeficiente c:");
        int c = EntradaTeclado.leInt();

        //Verificação do delta
        double deltaAux = (b*b) - 4*a*c;
        if(deltaAux < 0){
            System.out.println("Delta menor que 0");
            return;
        } 

        //Calculo
        double delta = Math.sqrt( deltaAux );
        double X1 = (delta - b)/2*a; 
        double X2 = (delta + b)/2*a;
        
        //Saida
        System.out.printf("X1 = %f \n", X1);
        System.out.printf("X2 = %f \n", X2);
    }
}

File: Exercitar python/Exercicio02.py
import math


#Declaração dos coeficientes
a = 0; b = 0; c = 0 

#Leitura dos coeficientes
print("Insira o valor de a:")
a = float(input())
print("Insira o valor de b:")
b = float(input())
print("Insira o valor de c:")
c = float(input())

#Considerar a = 0
if(a == 0):
    print("Não é possível dividr por 0\n (a = 0)")
    exit()

#Considerar delta < 0 
delta = (b*b) - (4*a*c)
if(delta < 0):
    print("Delta = 0, logo não existem raizes reais")
    exit()

#Calculo das raízes x1 e x2
x1 = ( (-b) + math.sqrt(delta) ) / (2*a)
x2 = ( (-b) - math.sqrt(delta) ) / (2*a)

#Saida 
print("X1 = {:.2f} X2 = {:.2f}".format(x1,x2))
